### DISCUSSION TITLE
Eligibility for github education for soon to be grad

### DISCUSSION THREAD

**Discussion by Chinwendu20** [community/community#49870] [GitHub Education]
Please I would like to know if I am still eligible for the education pack if I have completed my undergraduate work but I am yet to be conferred a degree because my final results are not out yet (it is taking quite a while because of some instability in my country). I still have access to my school's student portal and it shows that I am still a student.

**Comment by bkpecho** [comment]
Hi @Chinwendu20 

Yes, you may still be eligible for the education pack if you are still considered an enrolled student at your university. 

Best of luck with your application!
